Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -3656,6 +3656,10 @@ script:
     - description: The name of the virtual system to be configured. If no vsys is
         mentioned, this command will not use the vsys parameter.
       name: vsys
+    - description: The device group to target
+      name: device-group
```

Comment:
I used this so that the existing code doesn't have to be updated see

```
def initialize_instance(args: Dict[str, str], params: Dict[str, str]):
    global URL, API_KEY, USE_SSL, USE_URL_FILTERING, VSYS, DEVICE_GROUP, XPATH_SECURITY_RULES, XPATH_OBJECTS, \
        XPATH_RULEBASE, TEMPLATE, PRE_POST
    if not params.get('port'):
        raise DemistoException('Set a port for the instance')

    URL = params.get('server', '').rstrip('/:') + ':' + params.get('port', '') + '/api/'
    API_KEY = str(params.get('key')) or str((params.get('credentials') or {}).get('password', ''))  # type: ignore
    if not API_KEY:
        raise Exception('API Key must be provided.')
    USE_SSL = not params.get('insecure')
    USE_URL_FILTERING = params.get('use_url_filtering')

    # determine a vsys or a device-group
    VSYS = params.get('vsys', '')

    if args and args.get('device-group'):
        DEVICE_GROUP = args.get('device-group')  # type: ignore[assignment]
    else:
        DEVICE_GROUP = params.get('device_group', None)  # type: ignore[arg-type]

    if args and args.get('template'):
        TEMPLATE = args.get('template')  # type: ignore[assignment]
    else:
.
.
.
```

By passing the arg as `device-group` there is no need to update the code. It should be handled in the `initialize_instance` function.